<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-UKFMVAS" failures="1" tests="6" name="InvoiceTest" time="0.045" errors="0" timestamp="2025-06-18T00:06:25 GMT-07:00" skipped="5">
  <testcase classname="InvoiceTest" name="activateSim" time="0.026">
    <skipped/>
  </testcase> <!-- activateSim -->
  <testcase classname="InvoiceTest" name="activateSim" time="0.003">
    <skipped/>
  </testcase> <!-- activateSim -->
  <testcase classname="InvoiceTest" name="activateSim" time="0.003">
    <skipped/>
  </testcase> <!-- activateSim -->
  <testcase classname="InvoiceTest" name="activateSim" time="0.004">
    <failure message="expected:&lt;[]&gt; but was:&lt;[Login]&gt;" type="junit.framework.ComparisonFailure">
      <![CDATA[junit.framework.ComparisonFailure: expected:<[]> but was:<[Login]>
at junit.framework.Assert.assertEquals(Assert.java:85)
at junit.framework.Assert.assertEquals(Assert.java:91)
at InvoiceTest.activateSim(InvoiceTest.java:11)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:269)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:76)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- activateSim -->
  <testcase classname="InvoiceTest" name="activateSim" time="0.006">
    <skipped/>
  </testcase> <!-- activateSim -->
  <testcase classname="InvoiceTest" name="activateSim" time="0.003">
    <skipped/>
  </testcase> <!-- activateSim -->
</testsuite> <!-- InvoiceTest -->
